[2024-09-14T18:08:42.169+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather.save_city_info_weather manual__2024-09-14T18:03:40.437889+00:00 [queued]>
[2024-09-14T18:08:42.177+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather.save_city_info_weather manual__2024-09-14T18:03:40.437889+00:00 [queued]>
[2024-09-14T18:08:42.177+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-09-14T18:08:42.177+0000] {taskinstance.py:1284} INFO - Starting attempt 2 of 6
[2024-09-14T18:08:42.177+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-09-14T18:08:42.186+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): save_city_info_weather> on 2024-09-14 18:03:40.437889+00:00
[2024-09-14T18:08:42.191+0000] {standard_task_runner.py:55} INFO - Started process 2857 to run task
[2024-09-14T18:08:42.195+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'save_city_info_weather', 'save_city_info_weather', 'manual__2024-09-14T18:03:40.437889+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/saver_info.py', '--cfg-path', '/tmp/tmp52pebmm_']
[2024-09-14T18:08:42.196+0000] {standard_task_runner.py:83} INFO - Job 59: Subtask save_city_info_weather
[2024-09-14T18:08:42.235+0000] {task_command.py:389} INFO - Running <TaskInstance: save_city_info_weather.save_city_info_weather manual__2024-09-14T18:03:40.437889+00:00 [running]> on host 0c5fe11d0c9c
[2024-09-14T18:08:42.279+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=save_city_info_weather
AIRFLOW_CTX_TASK_ID=save_city_info_weather
AIRFLOW_CTX_EXECUTION_DATE=2024-09-14T18:03:40.437889+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-09-14T18:03:40.437889+00:00
[2024-09-14T18:08:42.280+0000] {sql.py:253} INFO - Executing: 
            create table if not exists weathers (
                id INTEGER PRIMARY KEY,
                max_temp VARCHAR(50),
                min_temp VARCHAR(50),
                wind VARCHAR(50),
                description VARCHAR(50)
            )
        
[2024-09-14T18:08:42.286+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-14T18:08:42.412+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-14T18:08:42.428+0000] {sql.py:364} INFO - Running statement: 
            create table if not exists weathers (
                id INTEGER PRIMARY KEY,
                max_temp VARCHAR(50),
                min_temp VARCHAR(50),
                wind VARCHAR(50),
                description VARCHAR(50)
            )
        , parameters: None
[2024-09-14T18:08:42.437+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=save_city_info_weather, task_id=save_city_info_weather, execution_date=20240914T180340, start_date=20240914T180842, end_date=20240914T180842
[2024-09-14T18:08:42.455+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-09-14T18:08:42.468+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
